### PR TITLE
Fix #245 & #211 & 206

### LINKED ISSUE #206: Change check icon into trash icon in plan-my-service component
Allow barman to remove a service directly inside the planning.
You have to change `check icon` into `trash icon` (https://material.io/icons/)

### LINKED ISSUE #211: Implement nbMax for services
In these components:

- Planning
- My service
- Fix my service

### LINKED ISSUE #245: Delete automatic `Active` field in barman creation
Allow user to create non active barman

### PR DESCRIPTION
**Pull request by kistora**
### Description of your PR

Will close #245 
Will close #211 
Will close #206 

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [ ] PR provide new tests for new behaviors
- [x] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [ ] Documentation have been updated if necessary

### CODE REVIEW

File: client/src/app/services/plan-my-services/plan-my-services.component.ts [hunk 1]
```diff
@@ -103,4 +103,8 @@ export class PlanMyServicesComponent implements OnInit {
     isPasted(service): Boolean {
         return new Service(service).isPasted();
     }
+
+    getColor(service: Service): Object {
+        return service.barmen ? service.barmen.length >= service.nbMax ? 'red' : 'green' : 'green';
```
Comment (Embraser01) [inline client/src/app/services/plan-my-services/plan-my-services.component.ts]: Simplify to only one ternary expression to
`(service.barmen && service.barmen.length >= service.nbMax) ? 'red' : 'green'`

File: client/src/app/services/planning/planning.component.ts [hunk 2]
```diff
@@ -45,4 +45,8 @@ export class PlanningComponent implements OnInit {
             });
         });
     }
+
+    getColor(service: Service): Object {
+        return service.barmen ? service.barmen.length >= service.nbMax ? 'red' : 'green' : 'green';
```
Comment (Embraser01) [inline client/src/app/services/planning/planning.component.ts]: Same here

**Comment by Embraser01** [comment]
For nbMax, we should also check in the back no?

**Comment by kistora** [comment]
No we don't. A barman can subscribe to a service even if the service is full.

**Pull request merged**